server/pages/index.pug

```diff
@@ -226,6 +226,56 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
+		script.
+			(function () {
+				// Mini tracks
+				function recordTracksEvent( eventName ) {
+					var statsPixel = 'http://pixel.wp.com/t.gif';
+					var stats = {
+						_pf: navigator.platform,
+						_lg: navigator.language,
+						_en: eventName,
+						_ht: window.screen.height,
+						_wd: window.screen.width,
+						_ut: 'anon',
+						_dl: document.location.origin + '/' + document.location.pathname,
+						_ts: Date.now(),
+						_tz: new Date().getTimezoneOffset() / 60
+					};
+
+					if ( window.currentUser ) {
+						stats[ '_ut' ] = 'wpcom:user_id';
+						stats[ '_ul' ] = currentUser.username;
+						stats[ '_ui' ] = currentUser.ID;
+					}
+
+					var pixel = document.createElement( 'img' );
+					pixel.src = statsPixel + '?' + Object.keys( stats ).map( key => key + '=' + encodeURIComponent( stats[ key ] ) ).join( '& ' );
+
+					var head = document.getElementsByTagName( 'head' )[ 0 ];
+					head.appendChild( pixel );
+
+					return stats;
+				}
+
+				try {
+					var pathName = window.location.pathname.split( "/" ).slice( 1, 2 )[ 0 ];
+					if ( pathName === 'log-in' && 'production' === window.configData.env ) {
+
+						var stats = recordTracksEvent( 'calypso_load_start' );
+						if ( ! window.calypsoLoadStartTime ) {
+							window.calypsoLoadStartTime = {};
+						}
+						window.calypsoLoadStartTime[ pathName ] = {
+							startTimestamp: stats['_ts'],
```

Comment:
this should be window.performance.now() if we want to measure elapsed time. Date.now reflects the system clock, which can change between the time this is recorded and the time that we finish loading. performance.now is divorced from the system clock and is more reliable.